### ISSUE TITLE
Glitches: Terranigma (EU) PAL video glitches when screen scrolls

### ISSUE THREAD

**Issue by treyturner** [SmokeMonsterPacks/Super-NT-Jailbreak#60]
## Expected behavior and actual behavior
Expected:
- No video glitches

Actual:
- Video glitches in the upper 1/8 of the screen while scrolling (seems OK inside rooms)
- Video of problem here: https://photos.app.goo.gl/oZcfC2MNk7XpDrwz1

## Steps to reproduce the problem
Start the game, go down, left, up (through the brown door) to talk to the elder, then exit the house (down, down), and travel around several screens once outside. The video I linked above shows moving left/right, but glitches also occur when moving up/down.

## ROM name and hash value
Happens in PAL mode (NTSC display) with:
Terranigma (Europe).sfc 93BA50D853E98E1CA227A2CA72389C0E3AC18D6B50C946B3F618C16C2D3EDD38

Also happens in NTSC mode (NTSC display) with:
Terranigma NTSC Patch (Hack) v1 BFeely.sfc EAA92D170305FA71CE2FCD5F1C0C65074BC95D97FE60B0AECFDED64DCE966558

## SD card model
Toshiba 2GB SD-M02G
SD card is pretty old school TBH, but I asked in the community chat and someone else reported problems as well, so I was asked to raise this issue

## Jailbreak firmware version: 
v6.5

## Relevant Super Nt settings
1080p, NTSC

**Comment by frederic-mahe** [comment]
thanks @treyturner , let's see if someone can confirm that bug.

**Comment by scotthammack** [comment]
I've only been able to make the visual glitches happen by setting the Super Nt to 1080p60 and PAL mode. However, my version of the NTSC fix ROM is different from the OP's.

Using the NTSC patched ROM, with the following Super Nt settings (applied before loading the ROM):
- 1080p60, NTSC mode: no glitching
**- 1080p60, PAL mode: glitching**
- 1080p50, NTSC mode: no glitching
- 1080p50, PAL mode: no glitching

Using the European ROM:
- 1080p60, NTSC mode: game refuses to boot (region lock screen), as expected
**- 1080p60, PAL mode: glitching**
- 1080p50, NTSC mode: region lock screen
- 1080p50, PAL mode: no glitching

I did not notice any audio dropouts during any of these tests.

SHA256 hashes: Terranigma (NTSC Fix).smc: 03318a9b1039420c567cf73e5edd9ba509cd17e1a049085dafd82dcfce8bc2bc
Terranigma (Europe).sfc: 93ba50d853e98e1ca227a2ca72389c0e3ac18d6b50c946b3f618c16c2d3edd38

**Comment by treyturner** [comment]
I came back to close this, because I discovered I was still set to PAL mode apparently (sorry!). Using the 'pure' NTSC patch provided by Great Hierophant (https://cdn.discordapp.com/attachments/418977246842454026/421546816056721419/Terranigma_NTSC.ips) and running the system in NTSC mode, the glitches are gone. I'll leave it up to contributors whether or not this should be help open; my issue is resolved.
